utils/ipfsJSONGenerator.js

```diff
@@ -0,0 +1,31 @@
+const basePath = process.cwd();
```

Comment:
I'd suggest to use ipfs-http-client npm module and generate whole metadata URI from scratch instead of having hardcoded CID here.
You can upload all images to ipfs, get CIDs per image, then construct separate metadata json for token in separate files (let's say 1, 2, 3, 4, 5 - each contain metadata for token 1, 2, 3, 4, 5), then upload a folder (containing token metadata JSONs) to ipfs, and use the returned CID as a base URI.

Reply:
please let me know if you have any question/concern about this